component-classes/index.js

```diff
@@ -555,4 +555,16 @@ export const backwardsCompatibleClasses = {
   radioDisabled: 'before:i-bg-$color-radio-background-disabled before:i-bg-$color-checkbox-background-disabled peer-checked:before:i-border-$color-radio-border-selected-disabled', //replaced in v1.5.0
   modalTitle: 'h-40 sm:h-48 items-center', // replaced by min-h-40 sm:min-h-48 items-start
   pillClose: 'pt-4 pb-6 text-m!' //replaced by py-10
+};
+
+export const pagination = {
+  currentPage: 'block md:hidden p-8 font-bold',
+  icon: 'flex w-16 h-16',
```

Comment:
Should the icon have a color as well? 🤔  According to the yaml file it should? 🤔 
```suggestion
  icon: 'flex w-16 h-16 s-icon',
```